source/documentation/businessandpropertyincome.html.md

```diff
@@ -357,6 +357,10 @@ This API should be used to submit any Annual Accounting Adjustments. The adjustm
 * [Retrieve a UK Property Business' Summary Adjustments](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-bsas-api/1.0#uk-property-business_retrieve-a-uk-property-business-summary-adjustments_get_accordion)
 * [Submit an Adjustment to a UK Property Business Summary](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-bsas-api/1.0#uk-property-business_submit-an-adjustment-to-a-uk-property-business-summary_post_accordion)
 
+|                      | Original (total of all 4 quarters) | 1<sup>st</sup> adjustment | Revised total | 2<sup>nd</sup> adjustment | Revised total |
```

Comment:
Looking at the description for the story and checking it locally I think the table is supposed to go in the index.md to match the description if you have a look at line 104 in the index.md. 

Also currently the note says `**From 30 November 2021 this API will be at V2.0, and V1.0 will no longer be available.**`  and does not match the description in the story which is `Adjustments are always made against the total of the original quarterly updates. Each new adjustment will overwrite the previous adjustment.`